extensions/amp-form/0.1/amp-form.js

```diff
@@ -701,36 +687,32 @@ export class AmpForm {
    * @private
    */
   templatesForSsr_() {
-    const successContainer =
-        this.form_.querySelector('div[submit-success]');
-    const errorContainer = this.form_.querySelector('div[submit-error]');
```

Comment:
The current validator behavior enforces that the `[submit-success]` message is a `div` element. If only the extension code is changed, forms will appear to be working correctly with `span` elements for example, but will fail validation. If we change it in the runtime code, I think we should change the validation code too.

https://github.com/ampproject/amphtml/blob/63050d54249734a34366b8acb34cb0dba0707e2e/validator/validator-main.protoascii#L4233-L4246

Reply:
Good point. I was just trying to make it consistent with existing code: https://github.com/ampproject/amphtml/blob/63050d54249734a34366b8acb34cb0dba0707e2e/extensions/amp-form/0.1/amp-form.js#L1064-L1065